src/status_im/contexts/keycard/error/view.cljs

```diff
@@ -0,0 +1,42 @@
+(ns status-im.contexts.keycard.error.view
```

Comment:
do not review this screen, is a temporary implementation will be finished in the next PRs